apps/skde/_posts/innhold/informasjonskapsler.md

```diff
@@ -0,0 +1,26 @@
+---
+filename: informasjonskapsler
+title: Informasjonskapsler
+lang: nb
+---
+
+Dette nettstedet benytter informasjonskapsler for å samle inn statistikk. Vi lagrer aldri informasjon som kan identifisere deg.
+
+Informasjonskapsler (cookies) er informasjonselementer som plasseres på datamaskinen din når du laster ned en nettside.
+
+Alle som besøker et nettsted skal ha informasjon om at nettstedet benytter informasjonskapsler. Du må godkjenne at informasjonskapsler blir benyttet. Godkjenningen gjøres gjennom innstillingene i din nettleser.
+
+Du kan hindre at slike informasjonskapsler lagres ved å endre på nettleserens innstillinger. Da vil imidlertid sidene ikke fungere optimalt. Du kan også slette informasjonskapsler i nettleseren eller ved filbehandling. [Slik administrerer du informasjonskapsler](https://nettvett.no/slik-administrer-du-informasjonskapsler/).
+
+Du har rett til å få vite om og godkjenne hvilke opplysninger som behandles, hva formålet med behandlingen er og hvem som behandler opplysningene, jf. ekomloven § 2-7b.
+
+### Informasjonskapsler i bruk
+
+Felles nettløsning for spesialisthelsetjenesten benytter informasjonskapsler til å samle inn anonymisert statistikk.
```

Comment:
Burde vi skrive SKDE i stedet for FNSP?

Reply:
Ja

Reply:
(kan bare slette den for helseatlas, hvis det ikke skal være noe der om personvern og informasjonskapsler)